.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx eslint ui/ --fix && npx prettier ui/ --write --loglevel warn
```

Comment:
Can we add the `prettier` command as a script in package.json so there's an easy way to run it _without_ committing?

Reply:
Done via your other suggest